src/linear_algebra/finite_dimensional.lean

```diff
@@ -140,6 +161,16 @@ lemma dim_eq_card : dim K V = fintype.card ι :=
 by rw [←h.mk_range_eq_dim, cardinal.fintype_card,
        set.card_range_of_injective (h.injective zero_ne_one)]
 
+/-- The findim of a vector space is equal to the cardinality of any basis.
+The assumption `[finite_dimensional K V]` is redundant, as there is a finite basis anyway, but it
+is necessary to make sense of `findim`. -/
+lemma findim_eq_card [finite_dimensional K V] : findim K V = fintype.card ι :=
```

Comment:
Would it be better to take the proof of `finite_dimensional K V` as an implicit argument here? This will mainly be used as a rewrite rule, right?

Reply:
I have even removed the argument completely, and proved it in the statement.